react/features/prejoin/functions.js

```diff
@@ -126,6 +171,79 @@ export function getDeviceStatusType(state: Object): string {
     return state['features/prejoin']?.deviceStatusType;
 }
 
+/**
+ * Returns the full dial out number (containing country code and +).
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {string}
+ */
+export function getFullDialOutNumber(state: Object): string {
+    const dialOutNumber = getDialOutNumber(state);
+    const country = getDialOutCountry(state);
+
+    return `+${country.dialCode}${dialOutNumber}`;
+}
+
+/**
+ * Returns the 'conferenceUrl' used for dialing out.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {string}
+ */
+export function getDialOutConferenceUrl(state: Object): string {
+    return `${getRoomName(state)}@${state['features/base/config'].hosts.muc}`;
```

Comment:
Maybe it's wise to extract the state tree name into a const and reuse that? You reference that a lot in this file.

Reply:
The thing is sometimes reusable selectors are more useful...